go.mod

```diff
@@ -32,6 +32,17 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require (
```

Comment:
Not sure why these ended up in a new block, but move them to a new block and `go mod tidy`.